src/main/kotlin/net/ccbluex/liquidbounce/features/command/commands/ingame/creative/CommandItemEnchant.kt

```diff
@@ -181,7 +181,8 @@ object CommandItemEnchant : CommandFactory, MinecraftShortcuts {
     }
 
     private fun creativeOrThrow(command: Command) {
-        if (mc.interactionManager?.hasCreativeInventory() == false) {
+        // TODO: you can probably have creative inventory in survival through abilities or something, "hack fix" I guess
+        if (mc.interactionManager?.gameMode?.isCreative == false) {
```

Comment:
```suggestion
        if (!player.isInCreativeMode) {
```
Used by minecraft in the parts that previously used the function. 

No need to check if the player is ingame as this is an ingame command.